### PR TITLE
Exclude failing tests only on GHA

### PR DESCRIPTION
**Pull request by andreaTP**
This way we can exclude specific tests only on GH Action

### CODE REVIEW

File: .github/workflows/scala3-build.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ jobs:
         run: |
           sbt -jvm-opts .jvmopts-ci \
             -Dakka.build.scalaVersion=3.0 \
-            -Dakka.test.tags.exclude=performance,timing,long-running \
+            -Dakka.test.tags.exclude=performance,timing,long-running,gh-eclude \
```
Comment (raboof) [inline .github/workflows/scala3-build.yml:32]:
```suggestion
            -Dakka.test.tags.exclude=performance,timing,long-running,gh-exclude \
```

File: akka-actor-tests/src/test/scala/akka/event/EventStreamSpec.scala [hunk 2]
```diff
@@ -315,7 +315,7 @@ class EventStreamSpec extends AkkaSpec(EventStreamSpec.config) {
       }
     }
 
-    "unsubscribe the actor, when it subscribes already in terminated state" in {
+    "unsubscribe the actor, when it subscribes already in terminated state" taggedAs GHExcludeTest in {
```
Comment (raboof) [inline akka-actor-tests/src/test/scala/akka/event/EventStreamSpec.scala:318]: Maybe we should link exclusions to tickets? In this case https://github.com/akka/akka/issues/30460

File: akka-actor-tests/src/test/scala/akka/io/UdpConnectedIntegrationSpec.scala [hunk 3]
```diff
@@ -125,7 +126,7 @@ class UdpConnectedIntegrationSpec extends AkkaSpec("""
     }
 
     // #26903
-    "be able to send and receive when server goes away (and comes back)" in {
+    "be able to send and receive when server goes away (and comes back)" taggedAs GHExcludeTest in {
```
Comment (raboof) [inline akka-actor-tests/src/test/scala/akka/io/UdpConnectedIntegrationSpec.scala:129]: (https://github.com/akka/akka/issues/30462)

File: akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala [hunk 4]
```diff
@@ -17,7 +18,7 @@ class LocalPubSubSpec extends ScalaTestWithActorTestKit with AnyWordSpecLike wit
 
   "A pub-sub topic running locally" must {
 
-    "publish to all local subscriber actors of a topic" in {
+    "publish to all local subscriber actors of a topic" taggedAs GHExcludeTest in {
```
Comment (raboof) [inline akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala:21]: (https://github.com/akka/akka/issues/30466)

File: akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingLeaseSpec.scala [hunk 5]
```diff
@@ -124,7 +124,7 @@ class ClusterShardingLeaseSpec(config: Config, rememberEntities: Boolean)
       testLease.setNextAcquireResult(Future.successful(true))
       expectMsg(3)
     }
-    "recover if lease lost" in {
+    "recover if lease lost" taggedAs GHExcludeTest in {
```
Comment (raboof) [inline akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingLeaseSpec.scala:127]: Did we see this be unstable on GHA? https://github.com/akka/akka/issues/30455 reports it on jenkins (where it only occurs rarely)

File: akka-remote/src/test/scala/akka/remote/artery/RemoteDeathWatchSpec.scala [hunk 6]
```diff
@@ -56,7 +56,7 @@ class RemoteDeathWatchSpec
 
   override def expectedTestDuration: FiniteDuration = 120.seconds
 
-  "receive Terminated when system of de-serialized ActorRef is not running" in {
+  "receive Terminated when system of de-serialized ActorRef is not running" taggedAs GHExcludeTest in {
```
Comment (raboof) [inline akka-remote/src/test/scala/akka/remote/artery/RemoteDeathWatchSpec.scala:59]: also seen (rarely) on jenkins so perhaps not necessary to exclude on GHA (https://github.com/akka/akka/issues/30454)

File: akka-stream-tests/src/test/scala/akka/stream/scaladsl/HubSpec.scala [hunk 7]
```diff
@@ -199,7 +199,7 @@ class HubSpec extends StreamSpec {
       downstream.expectComplete()
     }
 
-    "immediately cancel new producers while draining" in assertAllStagesStopped {
+    "immediately cancel new producers while draining" taggedAs GHExcludeTest in assertAllStagesStopped {
```
Comment (raboof) [inline akka-stream-tests/src/test/scala/akka/stream/scaladsl/HubSpec.scala:202]: https://github.com/akka/akka/issues/30464

**Review by raboof — APPROVED** [review]
LGTM, one (essential) typo, some we might not need to exclude, others it'd be nice to link to the issue

**Comment by raboof** [comment]
(needs a rebase)

**Review by ennru — APPROVED** [review]
LGTM, when the command line is corrected.
Linking to issues will be helpful.

File: akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala (hunk 4 above)
Comment (andreaTP) [reply akka-actor-typed-tests/src/test/scala/akka/actor/typed/pubsub/LocalPubSubSpec.scala:22]: done

**Review by andreaTP — COMMENTED** [review]
ready for next round @raboof 🙂

File: akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingLeaseSpec.scala (hunk 5 above)
Comment (andreaTP) [reply akka-cluster-sharding/src/test/scala/akka/cluster/sharding/ClusterShardingLeaseSpec.scala]: removed

File: akka-remote/src/test/scala/akka/remote/artery/RemoteDeathWatchSpec.scala (hunk 6 above)
Comment (andreaTP) [reply akka-remote/src/test/scala/akka/remote/artery/RemoteDeathWatchSpec.scala]: removed

**Review by raboof — APPROVED** [review]

**Pull request merged**